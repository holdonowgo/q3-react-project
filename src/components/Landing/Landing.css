.landing {
  font-size: 18px;
}
