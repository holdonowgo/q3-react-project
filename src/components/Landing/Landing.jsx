import React from 'react'
import { Header, Icon, Image, Container } from 'semantic-ui-react';
import './Landing.css';

const Landing = () => (
  <div>
    <Container>
    <Header as='h2' icon textAlign='center' size='huge'>
      <Icon name='bicycle' color='purple' />
      <Header.Content>
        Fitness Redifined for the 21st Century
      </Header.Content>
    </Header>
    <br/>
    <p className='landing'>Fit-Init is here to redifine the way you think about fitness.  From routines, to excersizes, to complex charts mapping your progress and training, fit init provides the solutions you need to get fit now!  Fit-Init Dashboard uses state of the art tracking technology and complex data mapping to give you all you need to formulate your ideal routine.  In addition we have routines, exercises and more to get you started.  Whether you want to loose 200lbs or get 1st place in that turkey trot 5k, fit-init provides the tools to get you there.  What are you waiting for?!</p>
    <br/>
    <Icon name='child' size='huge' color='teal'/>
  </Container>
  </div>
)

export default Landing
