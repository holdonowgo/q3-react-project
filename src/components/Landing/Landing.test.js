import React from 'react';
import ReactDOM from 'react-dom';
import renderer from 'react-test-renderer';
import { shallow, render } from 'enzyme';
import toJson from 'enzyme-to-json';

import Landing from './Landing.jsx';

describe('Search', () => {
  it('renders', () => {
    const div = document.createElement('div');
    ReactDOM.render(<Landing />, div);
  });

  test('snapshots', () => {
    const component = shallow(
      <Landing />
    );
    expect(toJson(component)).toMatchSnapshot();
    //
    // component.find('[name="toggle-preview"]').simulate('click');
    // expect(toJson(component)).toMatchSnapshot();
  });
})
