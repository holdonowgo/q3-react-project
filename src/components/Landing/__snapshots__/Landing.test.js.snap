exports[`Search snapshots 1`] = `
<div>
  <Container>
    <Header
      as="h2"
      icon={true}
      size="huge"
      textAlign="center">
      <Icon
        as="i"
        color="purple"
        name="bicycle" />
      <HeaderContent>
        Fitness Redifined for the 21st Century
      </HeaderContent>
    </Header>
    <br />
    <p
      className="landing">
      Fit-Init is here to redifine the way you think about fitness.  From routines, to excersizes, to complex charts mapping your progress and training, fit init provides the solutions you need to get fit now!  Fit-Init Dashboard uses state of the art tracking technology and complex data mapping to give you all you need to formulate your ideal routine.  In addition we have routines, exercises and more to get you started.  Whether you want to loose 200lbs or get 1st place in that turkey trot 5k, fit-init provides the tools to get you there.  What are you waiting for?!
    </p>
    <br />
    <Icon
      as="i"
      color="teal"
      name="child"
      size="huge" />
  </Container>
</div>
`;
